proto/viam/app/package/v1/package.proto

```diff
@@ -0,0 +1,106 @@
+syntax = "proto3";
+
+package viam.app.package.v1;
+
+import "google/api/annotations.proto";
+import "google/protobuf/struct.proto";
+
+option go_package = "go.viam.com/api/app/package/v1";
+
+service PackageService {
+  // Note: include HTTP annotations to these RPC definitions to utilize HTTP reverse
+  // proxies.
+
+  // CreatePackage uploads a package's contents and information to the cloud
+  rpc CreatePackage(stream viam.app.package.v1.CreatePackageRequest) returns (viam.app.package.v1.CreatePackageResponse) {
+    option (google.api.http) = {
+      post: "/package/v1/create_package"
+    };
+  }
+
+  // DeletePackage receives an authenticated request with the package name and array of versions and removes the associated packages
+  rpc DeletePackage(viam.app.package.v1.DeletePackageRequest) returns (viam.app.package.v1.DeletePackageResponse) {
+    option (google.api.http) = {
+      delete: "/package/v1/delete_package"
+    };
+  }
+
+  // GetPackages receives an authenticated request with the package name and version and, if present, generates returns the URL and metadata
```

Comment:
@agreenb wondering if comments, such as " // Primarily used by robot to download package files." from the scope doc should be included here

Reply:
Good question but nope, those were mostly used to describe the intent of each part of the scope.